exec/csnex/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+set(CSNEX
```

Comment:
This variable would be better named `CSNEX_SOURCES` (yes I know this was my mistake).

Reply:
Fixed.

Reply:
Fixed.